chatwoot/chatwoot
# Chore: Provide fixed attachment URLs for Channels

Prior to this change, The attachment URL sent from Chatwoot to 3rd party integrations like Whatsapp and Facebook
involved a 301 redirect before the original content is served. This causes intermittent breakages for the sent attachments.

fixes: #3632 
ref: https://blog.saeloun.com/2021/09/14/rails-7-adds-expiring-urls-to-active-storage.html

## QA instructions
We want to ensure that the attachments are rendered correctly on all the channels like SMS, WhatsApp, Facebook etc where the change has been made

## QA Completed

- [x] Tested WhatsApp, bandwidth SMS channels against s3 on staging
- [x]  Tested against local store to ensure download URLs properly render file without redirect
- [x]  Tested against azure store to ensure correct URLs are rendered
- [x]  Tested against s3 compatible using digital ocean spaces
